client/src/pages/Booking/useStyles.ts

```diff
@@ -0,0 +1,127 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    minHeight: '100vh',
+    background: '#fafafa',
+    '& .MuiInput-underline:before': {
+      borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
+    },
+
+    paddingTop: theme.spacing(12),
+  },
+  leftContainer: {
+    minHeight: '100%',
+    backgroundColor: 'transparent',
+    border: 'none',
+  },
+  rightContainer: {
+    minHeight: '100%',
+    border: 'none',
+    display: 'flex',
+    backgroundColor: 'transparent',
+    flexDirection: 'column',
+  },
+  nextBooking: {
+    display: 'none',
+    paddingTop: theme.spacing(1),
+    paddingRight: theme.spacing(10),
+    paddingLeft: theme.spacing(10),
+    '& .MuiCardHeader-title': {
+      fontWeight: 600,
+      fontSize: 12,
+      textTransform: 'uppercase',
+    },
+  },
+  bookingsBox: {
+    paddingTop: theme.spacing(2),
+    paddingRight: theme.spacing(10),
+    paddingLeft: theme.spacing(10),
+    '& .MuiCardHeader-title': {
+      fontWeight: 600,
+      fontSize: 12,
+      textTransform: 'uppercase',
+    },
+  },
+  nextBookingCardContent: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  currentBookingCardContent: {
+    display: 'flex',
```

Comment:
These styles can be applied to the Box component directly by passing them as props :)

Eg: `<Box display="flex" alignItems="center....`

Diito where applicable!

Reply:
done ✔️